content-src/asrouter/components/Button/_Button.scss

```diff
@@ -38,6 +38,10 @@
     &:active {
       background-color: $grey-90-30;
     }
+
```

Comment:
We tend to just use the hover state for stuff on new tab rather than a blue box shadow, but can you ask @aaronrbenson what he thinks?

Reply:
sure :)

Reply:
So the focus state is okay, I just made it match [these](https://design.firefox.com/photon/components/buttons.html)l; for the dismiss button I modified it so it can be focused by the keyboard without hovering the div (it just un-hides, basically).